### PR TITLE
terraform: Implement new command to get discovered versions

### PR DESCRIPTION
**Pull request by psibi**

### CODE REVIEW

File: clients/lsp-terraform.el [hunk 1]
```diff
@@ -222,6 +222,19 @@ This is a synchronous action."
...
 
+(defun lsp-terraform-ls-version ()
+  "Get information about the terraform binary version for the current module."
+  (interactive)
+  (let ((terraform-data (lsp-request
+                         "workspace/executeCommand"
+                         (list :command "terraform-ls.module.terraform"
+                               :arguments (vector (format "uri=%s" (lsp--path-to-uri (lsp-workspace-root)))))
+                         :no-wait nil
+                         :no-merge nil)))
```
Comment (yyoncho) [inline clients/lsp-terraform.el:233]: these are redundant.

File: clients/lsp-terraform.el [hunk 2]
```diff
@@ -222,6 +222,19 @@ This is a synchronous action."
...
+    (message "Required: %s, Current: %s"
```
Comment (yyoncho) [inline clients/lsp-terraform.el:234]: consider using `lsp--info`

**Review by yyoncho — APPROVED** [review]
Looks good, 2 minor nits:

File: clients/lsp-terraform.el (hunk 1 above)
Comment (psibi) [reply clients/lsp-terraform.el]: Fixed.

File: clients/lsp-terraform.el (hunk 2 above)
Comment (psibi) [reply clients/lsp-terraform.el]: Fixed, thanks!

**Review by jcs090218 — APPROVED** [review]

**Pull request merged**